opengrok-indexer/src/main/java/org/opengrok/indexer/util/CtagsUtil.java

```diff
@@ -29,10 +29,15 @@
 import org.opengrok.indexer.configuration.RuntimeEnvironment;
 import org.opengrok.indexer.logger.LoggerFactory;
 
+import java.io.File;
+import java.io.IOException;
```

Comment:
`UnusedImports` here and for `file.Files` (next line) per build

Reply:
fixed